### ISSUE TITLE
[Broken]az iot product test create

### ISSUE THREAD

**Issue by TiejunMS** [Azure/azure-cli#22137]
Hi, I run this command `az iot product test create --badge-type Pnp --at SymmetricKey --device-type FinishedProduct --models model` and the output is as below. The certificate of *certify.azureiotsolutions.com* is invalid. Could you take a look at it?

```
Command group 'iot product' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Error occurred in request., SSLError: HTTPSConnectionPool(host='certify.azureiotsolutions.com', port=443): Max retries exceeded with url: /deviceTests?GenerateProvisioningConfiguration=true&api-version=2020-05-01-preview (Caused by SSLError(CertificateError("hostname 'certify.azureiotsolutions.com' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net', '*.sso.azurewebsites.net'")))
Certificate verification failed. This typically happens when using Azure CLI behind a proxy that intercepts traffic with a self-signed certificate. Please add this certificate to the trusted CA bundle. More info: https://docs.microsoft.com/cli/azure/use-cli-effectively#work-behind-a-proxy.
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 93b879c9-53e4-253b-1d44-49bdaf975f53
* Version Independent ID: 166b7f9a-1095-aa54-758c-d66d9f817d4b
* Content: [az iot product test](https://docs.microsoft.com/en-us/cli/azure/iot/product/test?view=azure-cli-latest#az-iot-product-test-create)
* Content Source: [latest/docs-ref-autogen/iot/product/test.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/iot/product/test.yml)
* Service: **iot-hub**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
@TiejunMS Apologies for the late reply. Could you please check if you have any Proxy at your end ? If yes, try to follow to follow the steps mentioned [here](https://jiasli.github.io/azure-notes/cli/cli-proxy.html) and check if that helps.

Also to isolate the issue, could you try to run the same command completely from a different machine which is not in your corp network. Awaiting your reply.

**Comment by TiejunMS** [comment]
I can reproduce this issue both from corp network and my home. You can try to open https://certify.azureiotsolutions.com/ and the browser will also give an error.

**Comment by navba-MSFT** [comment]
@TiejunMS Thanks for getting back. I will check this further and get back to you once I have more details on this.

**Comment by navba-MSFT** [comment]
@TiejunMS I am discussing this with the Product Owners. I am currently waiting for their response.

**Comment by c-ryan-k** [comment]
Just a note - we have a PR in progress to fix this from the `azure-iot` extension:
https://github.com/Azure/azure-iot-cli-extension/pull/514

It looks like the old endpoint at `certify.azureiotsolutions.com` is no longer active or service requests.

In the meantime (until this release goes out and is live on the extension index) - a workaround would be to append:
`--base-url https://prod.certsvc.trafficmanager.net`
to each `az iot product` command to send it to the correct endpoint.

**Comment by navba-MSFT** [comment]
Thanks @c-ryan-k .

@TiejunMS I hope you had a chance to look at the above comment. Please let us know if you have any follow-up questions on this. Awaiting reply.

**Comment by TiejunMS** [comment]
Thanks both! I have verified the command `az iot product test create` is fixed with proposed workaround. But the command `z iot product test task create --type QueueTestRun` will be stuck for more than 20 minutes. When it worked before, this command would be completed within 1 minute. Do you know if this issue is related?

**Comment by c-ryan-k** [comment]
@TiejunMS - we've just released [`v0.14.1`](https://github.com/Azure/azure-iot-cli-extension/releases/tag/v0.14.1) - and that version should be available on the extension index shortly. Regarding the service issue - I don't believe the issues are related, but we might need to route that to the AICS service team to find out more information.

**Comment by navba-MSFT** [comment]
@c-ryan-k Thanks for looking into this. 

@TiejunMS As mentioned in the above comment, Please update the IOT extension and that should fix your actual issue. Regarding your issue with az iot product test task create --type QueueTestRun command causing a hang, is unrelated to this issue. To address the hang issue, I am looping you with the product owners offline. They would need a few details to troubleshoot it further. With that I am proceeding with the closure of this github thread. Please get back to me if you have any queries.
